access-control-app/docs/api.md

```diff
@@ -459,7 +455,7 @@ For system, provider, and single instance identities, the grantable permissions
 
 ### <a name="search-acls"></a> Search ACLs
 
-ACLs can be searched for by sending a GET request to `%CMR-ENDPOINT%/acls`, or a POST to `%CMR-ENDPOINT%/acls/search`. A valid Echo token is required to search ACLs.  If no token is supplied then guest permissions are used.
+ACLs can be searched for by sending a GET request to `%CMR-ENDPOINT%/acls`, or a POST to `%CMR-ENDPOINT%/acls/search`. A valid EDL bearer token is required to search ACLs.  If no token is supplied then guest permissions are used.
```

Comment:
NITPIC: Double spaces after a period is no longer needed with variable width fonts, and the usage is not even consistent within this block of text.